parquet-column/src/main/java/org/apache/parquet/filter2/predicate/FilterApi.java

```diff
@@ -72,26 +73,50 @@ public static IntColumn intColumn(String columnPath) {
     return new IntColumn(ColumnPath.fromDotString(columnPath));
   }
 
+  public static IntColumn intColumn(Type.ID id) {
```

Comment:
@huaxingao and @shangxinli, has the Parquet community considered using Iceberg expressions and filters? I know that's a separate change, but it would be a great way to pick up a cleaner filter API that handles expression binding.